test/servico/cst03Test.java

```diff
@@ -53,30 +51,34 @@ public void tearDown() {
     }
     
     @Test
-    public void testeNotificaçãoDeAusência_SemMotivo() throws ParseException
-    {
+    public void testeNotificaçãoDeAusência_SemMotivo() throws ParseException{
+        //Login com usuário "Funcionario"
         LoginService loginService = new LoginService();
-        assertTrue(loginService.VerificarUsuarioESenha("Funcionario1", "123456"));
+        assertTrue(loginService.VerificarUsuarioESenha("Funcionario", "123456"));
         
         ProfessorService professorService = new ProfessorService();
         ProfessorModel professor = professorService.obterProfessorPorNome("Professor1");
         
+        //Navegação de "Notificação de ausência" 
         NotificacaoService notificaçãoService = new NotificacaoService();
         List<AusenciaModel> ausencias = notificaçãoService.listarAusencias();        
         assertEquals(0, ausencias.size());
         
         Boolean exceptionOk = false;
         
-        try
-        {
-            notificaçãoService.notificarAusencia(professor.id, "25/11/2013 20:01", "25/11/2013 21:29", "", new LinkedList<String>());
+        try{
+            //notifica ausência do Professor
+            notificaçãoService.notificarAusencia(professor.id, "13/06/2016 20:01", "13/06/2016 21:29", "", new LinkedList<String>());

```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 80 characters (found 133).](https://www.codacy.com/app/calebepb/substituicao/file/2738690362/issues/source?bid=3308597&fileBranchId=3353674#l71)